liubai-docs/docs/guide/markdown-support.md

```
@@ -0,0 +1,72 @@
+
+留白记事支持简单的 Markdown 语法：
+
+### 行内代码
+
+在编辑器上，输入 \` + 你的行内文字 + \` 即可包裹行内代码。
+
+![inline code](./assets-md-support/01.png)
+
+### 粗体
+
+在编辑器上，输入 ** + 你的粗体文字 + ** 即可包裹粗体。
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers
null

(MD037, no-space-in-emphasis)